CHANGELOG.md

```diff
@@ -9,15 +9,13 @@ ENHANCEMENTS:
 * **netapp-ontap_storage_volume_snapshot_resource**: Add support for import ([#42](https://github.com/NetApp/terraform-provider-netapp-ontap/issues/42))
```

Comment:
Look like in a previous change we didn't merge the change long right.

Can you delete lines 4, 5, 6, 7, 8, 9. I think that removes all the duplicates.